### ISSUE TITLE
Android Auto 

### ISSUE THREAD

**Issue by jolkozak** [polymorphicshade/NewPipe#368]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/polymorphicshade/NewPipe/issues) or [closed](https://github.com/polymorphicshade/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/polymorphicshade/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Hi,

Could you please add option to use NewPipe in Android Auto?
Please check: https://github.com/TeamNewPipe/NewPipe/pull/9592

### Why do you want this feature?

Now NewPipe can only be used using AnyAutoAudio

### Why ist the feature relevant to this fork?

The original application is not interested in making these changes.

**Comment by HatakeKakashri** [comment]
> The original application is not interested in making these changes.

I don't think this is true. NewPipe has an upcoming major redesign. Although that is only code level. This feature is not out of the scope as such even tbough the PR has not been very active.
